conf/otherassetsdisposal.routes

```diff
@@ -91,6 +91,11 @@ GET         /:srn/submit-check-answers-asset-disposal/view/:year/:current/:previ
 
 GET         /:srn/asset-disposal-list/:page                                                                    controllers.nonsipp.otherassetsdisposal.ReportedOtherAssetsDisposalListController.onPageLoad(srn: Srn, page: Int, mode: Mode = NormalMode)
 POST        /:srn/asset-disposal-list/:page                                                                    controllers.nonsipp.otherassetsdisposal.ReportedOtherAssetsDisposalListController.onSubmit(srn: Srn, page: Int, mode: Mode = NormalMode)
+POST        /:srn/change-asset-disposal-list/:page                                                            controllers.nonsipp.otherassetsdisposal.ReportedOtherAssetsDisposalListController.onSubmit(srn: Srn, page: Int, mode: Mode = ViewOnlyMode)
```

Comment:
Is there a reason why this url has `change-` at the beginning? It doesn't seem to be related to `CheckMode`, so I'm a bit confused.